pkg/license/identifier.go

```diff
@@ -19,58 +19,48 @@ package license
 
 import (
 	"fmt"
-	"io/fs"
 	"path/filepath"
 	"regexp"
-	"strings"
 	"sync"
 
+	"github.com/google/licensecheck"
+
 	"github.com/apache/skywalking-eyes/assets"
 	"github.com/apache/skywalking-eyes/internal/logger"
 )
 
-var licenseTemplatesDir = "lcs-templates"
+const (
+	// coverageThreshold is the minimum percentage of the file that must contain license text.
+	// Reference: https://github.com/golang/pkgsite/blob/d43359e3a135fc391960db4f5800eb081d658412/internal/licenses/licenses.go#L48
+	coverageThreshold = 75
```

Comment:
And we might expose this as a configuration(with 75 as default) for advanced users?

Reply:
Ok, but this requires to change a serials of functions, I will file another PR to do this.

Reply:
> Ok, but this requires to change a serials of functions, I will file another PR to do this.

Sure